rules_building_block/defense_evasion_invalid_codesign_imageload.toml

```diff
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2023/09/27"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/09/27"
+
```

Comment:
```suggestion
bypass_bbr_timing = true

```

Reply:
It is not needed since we use the default BBR timing here
![image](https://github.com/elastic/detection-rules/assets/26856693/0c8df422-0a09-43af-8e93-31d901c766d6)